### PR TITLE
Fix/add BlobReader comments

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@nguerrera

**Comment by tmat** [comment]
@nguerrera Could you take a look? This is a change in master.

File: src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobReader.cs [hunk 1]
```diff
@@ -435,10 +435,11 @@ private static void ThrowInvalidSerializedString()
         /// <summary>
         /// Reads type code encoded in a serialized custom attribute value. 
         /// </summary>
+        /// <returns><see cref="SerializationTypeCode.Invalid"/> if the encoding is invalid.</returns>
```
Comment (stephentoub) [inline src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobReader.cs]: If we have a <returns/> comment, we probably want it to say what it returns not just in the case of error, right?

**Comment by stephentoub** [comment]
One minor comment, but overall LGTM.  I assume at some point we'll go through and polish up the docs in general, for content, for consistency, for grammar, etc.?

File: src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobReader.cs (hunk 1 above)
Comment (nguerrera) [inline src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobReader.cs]: Right.

**Comment by nguerrera** [comment]
I agree with the minor comment, but merging this since it's strictly more informative than what we had. 

We should indeed do a full pass over these docs.

**Pull request merged**